Rock-Paper-Scissor Game/README.md

```diff
@@ -0,0 +1,24 @@
+# Rock-Paper-Scissor Game 🗿 📜 ✂️
+
+
+*This Rock-Paper-Scissor Game allows you to play rock paper scissor with the computer. Have fun playing :)*
+
+> Used Technologies
+- HTML
+- CSS
+- JAVASCRIPT
+
+
+### Steps to use: 
+
+- Download or clone the repositor
+`
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+`
+
+- Go to the directory
+- Run the index.html file
+- Start Playing!!!
+
+<!-- image -->
+
```

Comment:
ok, am merging this project but upload a screenshot image here with a new PR for it ! 👍

Reply:
@Ayushparikh-code Okay.